app/javascript/shared/http.ts

```diff
@@ -33,11 +33,18 @@ if (csrfToken) {
 }
 
 const kyApi = ky.extend({
+  headers: {
+    'Content-Type': 'application/json',
+  },
```

Comment:
Ensure that all of these requests are made as JSON requests.

I think this actually isn't super necessary, because I think that all API routes are automatically considered JSON, anyway (per https://github.com/davidrunger/david_runger/blob/9ddd20b503420b9ca59c0834e2709b4426fbada4/config/routes.rb#L69), but it's still probably a good idea to put this here, even if it's redundant, for now.